tests/Functional/Form/CheckboxUrlLabelFormTypeTest.php

```diff
@@ -29,9 +29,13 @@ public function setUp()
     public function testFormTwigTemplate()
     {
         $twig = $this->getContainer()->get('twig');
-        if (class_exists(ContainerAwareRuntimeLoader::class)) {
+        if (class_exists('Symfony\Component\Form\FormRenderer')) {
+            $extension = $twig->getExtension(FormExtension::class);
+            $extension->initRuntime($twig);
+            $renderer = $extension->renderer;
+        } elseif (class_exists('Symfony\Bridge\Twig\Form\TwigRenderer')) {
```

Comment:
please use the ::class constant

Reply:
oh, also, this is in a test. do we still need all cases, or was that to support old symfony versions that we don't support anymore?

Reply:
I must admit, that i am curently lost which is the right way for which version.